### PR TITLE
Contact

### PR DESCRIPTION
**Pull request by ralberd**

### CODE REVIEW

File: regression/GradientCheck/GC_Analyze_CompliantContact/confirm_gradient_convergence_trend.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import sys
```
Comment (rawildman) [inline regression/GradientCheck/GC_Analyze_CompliantContact/confirm_gradient_convergence_trend.py:1]: Are these `confirm_gradient_convergence.py` and `finite_difference_utils.py` files the same? If so, would you consider moving them up a directory, maybe make a `GradientCheck/utils` directory for these?

File: regression/GradientCheck/GC_Analyze_DfDX_CompliantContact/.logamgx [hunk 2]
```diff
@@ -0,0 +1,116 @@
+AMGX_LOG	RUNTIME_VERSION=10.2
```
Comment (rawildman) [inline regression/GradientCheck/GC_Analyze_DfDX_CompliantContact/.logamgx:1]: Are these log files needed? I haven't seen them before so not sure if they're leftover from a previous run.

File: regression/GradientCheck/GC_Analyze_CompliantContact/confirm_gradient_convergence_trend.py (hunk 1 above)
Comment (ralberd) [reply regression/GradientCheck/GC_Analyze_CompliantContact/confirm_gradient_convergence_trend.py:1]: The confirm_gradient_convergence.py file is what Adam wrote for the GC tests. I'm pretty sure that doesn't change from test to test but I could be wrong. The finite_difference_utils.py file I can move up a directory.

File: regression/GradientCheck/GC_Analyze_DfDX_CompliantContact/.logamgx (hunk 2 above)
Comment (ralberd) [reply regression/GradientCheck/GC_Analyze_DfDX_CompliantContact/.logamgx]: I removed them

File: regression/GradientCheck/GC_Analyze_CompliantContact/confirm_gradient_convergence_trend.py (hunk 1 above)
Comment (ralberd) [reply regression/GradientCheck/GC_Analyze_CompliantContact/confirm_gradient_convergence_trend.py:1]: I moved finite_difference_utils.py to a utils directory

**Review by rawildman — APPROVED** [review]

**Pull request merged**